# HG changeset patch
# User kevinw
# Date 1282924624 -3600
# Node ID 21c29458b334f7b1f05a185442ab37ab8de1ea7b
# Parent  8e5955ddf8e491a54b7e9446424fdee02e6950c8
6980392: TEST_BUG: gc/6581734/Test6581734.java has typo
Summary: simple correction in testcase
Reviewed-by: mchung

diff -r 8e5955ddf8e4 -r 21c29458b334 test/gc/6581734/Test6581734.java
--- openjdk.old/hotspot/test/gc/6581734/Test6581734_.java	2010-10-08 22:29:24.000000000 +0200
+++ openjdk/hotspot/test/gc/6581734/Test6581734.java	2010-11-11 15:47:15.000000000 +0100
@@ -121,7 +121,7 @@
         }
 
         if (collectorsWithTime<collectorsFound) {
-            throw new RuntimeException("collectors found with zero time";
+            throw new RuntimeException("collectors found with zero time");
         }
         System.out.println("Test passed.");
     }
